src/react-components/src/components/00-base/SectionName.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+
+const SectionName = ({ children }) => <div className={bem('main', ['research'])} >{children}</div>;
```

Comment:
This seems pretty specific - does it need to be its own component?  If it's supposed to only be used by the ResearchNow hero, can we wrap the ResearchNow Hero Story with  
<div className={bem('main', ['research'])} ></div> ?

Reply:
It will be generic and used for all of the SectionNames stipulated here: https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Master?node-id=674%3A274

See former thread: https://github.com/NYPL/nypl-design-system/pull/106/files#r401805076

Reply:
ah yes my memory is jogged now, sorry about that.